tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/Selenium/IWebDriverProxy.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="IWebDriverProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.Selenium;
+
+/// <summary>
+/// DuckTyping interface for OpenQA.Selenium.WebDriver
+/// </summary>
+internal interface IWebDriverProxy : IDuckType
+{
+    /// <summary>
+    /// Gets the <see cref="SessionId"/> for the current session of this driver.
+    /// </summary>
+    object? SessionId { get; }
+
+    /// <summary>
+    /// Gets the capabilities of the current driver.
+    /// </summary>
+    ICapabilities Capabilities { get; }
+
+    /// <summary>
+    /// Calls method: System.Object OpenQA.Selenium.WebDriver::ExecuteScript(System.String,System.Object[])
+    /// </summary>
+    [Duck(ParameterTypeNames = new string[] { "System.String", "System.Object[]" })]
```

Comment:
```suggestion
    [Duck(ParameterTypeNames = ["System.String", "System.Object[]"])]
```

Reply:
done in [1bf3e94](https://github.com/DataDog/dd-trace-dotnet/pull/5505/commits/1bf3e943771e108583d46b35b6c01295939d844a)